app/views/_includes/applications/course-details.njk

```diff
@@ -1,46 +1,67 @@
-  <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-7 govuk-!-margin-bottom-2 ">Course applied for</h2>
-      {{ govukSummaryList({
-        rows: [{
-          key: {
-            text: "Training provider"
-          },
-          value: {
-            text: application["provider"]
-          }
-        }, {
-          key: {
-            text: "Course"
-          },
-          value: {
-            text: application["course"]
-          }
-        }, {
-          key: {
-            text: "Full time or part time"
-          },
-          value: {
-            text: application.studyMode
-          }
-        }, {
-          key: {
-            text: "Location"
-          },
-          value: {
-            html: application.location + '<br> SCIL, Dillington House, Ilminster, Somerset, TA19 9DT '
-          }
-        }, {
-          key: {
-            text: "Accredited body"
-          },
-          value: {
-            html: application.accreditedBody
-          }
-        } if application.accreditedBody != application.provider, {
-          key: {
-            text: "Funding type"
-          },
-          value: {
-            text: application.fundingType
-          }
-        }]
-      }) }}
+{% set locationHtml %}
+<ol class="govuk-list">
```

Comment:
Not usre it should be an OL and LI

Thinking just line breaks and a para maybe an address tag?

Reply:
I've changed to line breaks and a paragraph tag.